### PR TITLE
feat: add support for injectScript

### PR DESCRIPTION
**Pull request by mbrevda**
Noticed the option was missing, would be great to get this added!

### CODE REVIEW

**Comment by mbrevda** [comment]
Hey - any way this can be prioritized?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "webpagetest",
-  "version": "0.5.0",
+  "version": "0.5.1",
```
Comment (tkadlec) [inline package.json:3]: Actually, let's not bump the version yet, @mbrevda. I'll throw in one other param we wanted to add and then bump the version.

**Comment by mbrevda** [comment]
sure, here you go!

**Review by tkadlec — APPROVED** [review]
Looks great! Thanks @mbrevda!

**Comment by tkadlec** [comment]
Whoops! One more quick change, @mbrevda to fix the tests and then we can merge.

Looks like we need to update the fixture here: https://github.com/WebPageTest/webpagetest-api/blob/5889c0d3ecdad071bfdde59b3c525548fe1acf38/test/fixtures/command-line/help-test.txt#L35

So that it uses `<injectScript>` instead of `<script>`

**Comment by mbrevda** [comment]
like that?

**Comment by tkadlec** [comment]
Perfect! Thanks @mbrevda!

**Comment by mbrevda** [comment]
Pleasure - eta on version bump and github action update to use this?

**Comment by mbrevda** [comment]
Any ETA on when this will be published?

**Comment by mbrevda** [comment]
ping - can we get this published please?

**Comment by tkadlec** [comment]
Done! Sorry it took a bit. We'll have to roll the other updates out in another release, but this way we get your updates out the door. Thanks for doing this!

**Comment by mbrevda** [comment]
Awesome, thanks! Can we please bump the github action to use it? https://github.com/WPO-Foundation/webpagetest-github-action/pull/9

**Pull request merged**